library/src/main/java/com/novoda/simplechromecustomtabs/SimpleChromeCustomTabs.java

```diff
@@ -20,42 +19,35 @@
 
 public final class SimpleChromeCustomTabs implements WebNavigator, Connection, AvailableAppProvider {
 
-    private static Context applicationContext;
     private Connection connection;
     private WebNavigator webNavigator;
     private AvailableAppProvider availableAppProvider;
 
-    private SimpleChromeCustomTabs() {
-        //no-op
-    }
-
-    private static class LazyHolder {
-        private static final SimpleChromeCustomTabs INSTANCE = new SimpleChromeCustomTabs();
+    /**
+     * @deprecated There is no need to call it anymore in order to use this library,
```

Comment:
I wouldn't recommend to keep calling this method because forces the user to extend from `Application` and so adding more code from the client's perspective when is not needed. Keeping it would not tell the user that this is not need anymore so that they can remove code from their side. Maybe you want to remove the deprecated annotation but keep the comment?